webpack.config.js

```diff
@@ -0,0 +1,68 @@
+import path from 'path';
+import webpack from 'webpack';
+import UglifyJSPlugin from 'uglifyjs-webpack-plugin';
+
+import {settings} from './gulp/config';
+
+const dev = {
+	entry: {
+		polyfills: './src/app/polyfills.js',
+		app: './src/app/index.js',
+		vendor: settings.sources.externalJs
+	},
+	output: {
+		path: path.join(__dirname, settings.destinations.dev.app),
+		filename: '[name].bundle.js'
+	},
+	plugins: [
+		new webpack.optimize.CommonsChunkPlugin({
+			name: ['app', 'vendor', 'polyfills']
+		}),
+		new webpack.ProvidePlugin({
+			$: 'jquery',
+			jQuery: 'jquery'
+		})
+	],
+	module: {
+		rules: [
+			{
+				test: /\.js$/,
+				exclude: /node_modules/,
+				use: {
+					loader: 'babel-loader'
+				}
+			}
+		]
+	}
+};
+
+const prod = {
+	entry: {
+		...dev.entry
+	},
+	output: {
+		path: path.join(__dirname, settings.destinations.prod.app),
+		filename: '[name].bundle.min.js'
+	},
+	plugins: [
+		...dev.plugins,
+		new UglifyJSPlugin({
+			uglifyOptions: {
+				compress: {
+					drop_console: true, // eslint-disable-line camelcase
+					drop_debugger: true // eslint-disable-line camelcase
+				}
+			}
+		}),
+		new webpack.DefinePlugin({
```

Comment:
First I thought it would make sense to expose the DefinePlugin settings to the `package.json` like:

```
"baumeister": {
  "globalConstants": {}
}
```

But I guess it’s better to handle it like:

```json
"baumeister": {
  "webpack": {
    "DefinePlugin": {
      "dev": {},
      "prod": {
        "process.env": {
          "NODE_ENV": "'production'"
        }
      }
    },
    "ProvidePlugin": {
      "$": "jquery",
      "jQuery": "jquery"
    }
  }
}
```
What do you think @nicolaisueper @phlmn